docs/anonymizer/adding_anonymizers.md

```diff
@@ -0,0 +1,19 @@
+# Supporting new types of PII anonymizers
+
+Presidio anonymizer can be easily extended to support additional anonnymization methods.
+
+## Extending the anonymizer for additional PII anonymizations:
+
+1. Under the path presidio_anonymizr/anonymizers
+2. Create a new file based on `Anonymizer`
+3. Implement the methods: 
+    - `anonymize` - gets the data and returns a new text expected to replace the old one.
+    - `validate` - validate the parameters entered for the anonymizer exists and valid.
+    - `anonymizer_name` - this method helps to automatically load the existing anonymizers.
+4. Restart the anonymizer.
```

Comment:
Because of the mechanism we have for finding all subclasses of `Anonymizer`, the user needs to make his/her new class visible somehow, e.g. by putting it in `__init__.py`

Reply:
RIGHT!

Reply:
Added